index.ts

```diff
@@ -68,18 +68,21 @@ async function get<TValue extends Value>(key: string): Promise<TValue | undefine
 }
 
 async function set<TValue extends Value>(key: string, value: TValue, maxAge: TimeDescriptor = {days: 30}): Promise<TValue> {
```

Comment:
I think the only issue here is that `undefined` isn't accepted by the types

```suggestion
async function set<TValue extends Value>(key: string, value: TValue | undefined, maxAge: TimeDescriptor = {days: 30}): Promise<TValue> {
```

I released a prerelease 4.3.0-0 if you want to try it in RGH in the original issue that discussed this